keep-libruntime/fixtures/return_1.wkld

```diff
@@ -0,0 +1,8 @@
+{
+  "modules": {
+    "return_1": [0,97,115,109,1,0,0,0,1,133,128,128,128,0,1,96,0,1,127,3,130,128,128,128,0,1,0,4,132,128,128,128,0,1,112,0,0,5,131,128,128,128,0,1,0,1,6,129,128,128,128,0,0,7,145,128,128,128,0,2,6,109,101,109,111,114,121,2,0,4,109,97,105,110,0,0,10,138,128,128,128,0,1,132,128,128,128,0,0,65,1,11]
+  },
+  "function_module":"return_1",
+  "function_name":"main",
```

Comment:
how about `main::return_1` ? instead of splitting it?
Or:
```json
"function" : {
   "module": "return_1",
   "name": "main",
   "args": []
}
```

Reply:
I'm not convinced that it's worth to split this out into a separate structure, but I'm also not opposed, so let's try it.